flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorImpl.java

```diff
@@ -126,6 +162,15 @@ public ChannelStateWriter getChannelStateWriter() {
 		return channelStateWriter;
 	}
 
+	@VisibleForTesting
+	int getAbortedCheckpointSize() {
+		return abortedCheckpointIds.size();
+	}
+
+	private boolean checkpointAlreadyAborted(long checkpointId) {
```

Comment:
~~nit~~ ambiguous name (not `nit` because I indeed suggested a wrong name ))

`getAndClearAbortedStatus`?

Reply:
I think `getAndClearAbortedStatus` did not give explicit description of the result when checking the notified checkpoint-id. I think `isCheckpointAlreadyAborted` might be better than original name. 

Moreover, I think we don't need to explain that we will clear the aborted checkpoint-id if found which is too technically.

Reply:
By convention, `isXxx` should be used for boolean getters.
I think it's always good to give self-explaining names to methods to prevent surprises and looking into their code.

Reply:
How about `checkAndClearAbortedStatus` as I need to return boolean result.

Reply:
:+1: